api/src/main/java/org/apache/iceberg/TableScan.java

```diff
@@ -171,3 +171,4 @@ default TableScan select(String... columns) {
    */
   boolean isCaseSensitive();
 }
+
```

Comment:
Nit: I think this is the only change to this file, so it would be good to revert it and reduce the number of files touched in the PR.